workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -47,7 +47,7 @@ PlayerSettings:
   defaultScreenWidthWeb: 960
   defaultScreenHeightWeb: 600
   m_StereoRenderingPath: 0
-  m_ActiveColorSpace: 1
```

Comment:
Is this intentional?

Reply:
Yes, without this change either iOS Sim or iOS device (don't remember which exactly) doesn't work correctly.

Reply:
👍